examples/metal-2-fabric-aws-connection/main.tf

```diff
@@ -2,6 +2,12 @@ provider "equinix" {
   client_id     = var.equinix_client_id
```

Comment:
The example folder should be named `service-token-metal-2-fabric-aws-connection` to stay consistent with example names starting with the submodule they are using for the connection.